core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -555,6 +569,18 @@ public InputStream readAttachment(Attachment a) throws FileSystemException, IOEx
 		return binaryInputStream;
 	}
 
+	@Override
+	public EmailMessage getFileFromAttachment(Attachment a) throws FileSystemException {
+		if (a instanceof ItemAttachment) {
+			ItemAttachment ia = (ItemAttachment) a;
+			Item aItem = ia.getItem();
+			if (aItem instanceof EmailMessage) {
+				return (EmailMessage) aItem;
+			}
+		}
+		return null;
```

Comment:
Hier ook een log.warn oid neerzetten dat een onverwacht attachment ding opgegeven is?